pyscriptjs/tests/common.py

```diff
@@ -0,0 +1,29 @@
+"""Reusable testing functionality"""
+
+import re
+
+# Regexes
+RE_PROTOCOL = re.compile(r"(https?:)\/+")
+RE_SLASHES = re.compile(r"\/+")
+
+# Test timing (how long to wait for and how often to check page events)
+MAX_TEST_TIME = 30  # Number of seconds
+TEST_TIME_INCREMENT = 0.25  # 1/4 second
+
+
+def _url_join(base, *parts):
```

Comment:
I think I'm missing the context here.... what's the use case for this?  Anything else other than just join parts of an URL? If not, I'd recommend just using standard python methods like urlparse.urljoin or even "/".join ...